pkgs/development/python-modules/httpagentparser/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "httpagentparser";
+  version = "1.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "73Y9MZk912GCWs7myLNL4yuVzxZ10cc8PNNfnlKDGyY=";
+  };
+
+  checkInputs = [
+    tox
```

Comment:
We do not want to run the tests through tox because it can cause issues and is harder to debug. Please run pytest via pytestCheckHook.